week2/homework/src/index.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
 // TODO: Write the homework code in this file
+const yargs = require('yargs');
+const commands = require('./commands');
+
+let command = yargs.argv._[0];
+let todo = yargs.argv.todo;
```

Comment:
your solution with `help` option is just to make the following and that will disable it's default value

**Option#1**

you may need to have a new variable to use it's returned value in which it turns off the help command
```javascript
const y = yargs.help(false);

let command = y.argv._[0];
let todo = y.argv.todo;
```

**Option#2**

or you may include it at line 4 as:
```javascript
const yargs = require('yargs').help(false);
```

Since you are not changing their direct value: `command` & `todo` should be a `const` too.